.github/workflows/pr-receive.yaml

```diff
@@ -111,6 +111,7 @@ jobs:
         with:
           name: pr
           path: ${{ env.PR }}
+          overwrite: true
```

Comment:
The default here is false, so this will change the upload behaviour. Is that what we want?